### ISSUE TITLE
sugg2

### ISSUE THREAD

**Issue by jesuisjoszi** [Pbtm999/pb-boombox#2]
and a question whether there will be an update that will allow, for example, to raise the prop (I saw something similar in wasabi_boombox), thank you in advance

**Comment by Pbtm999** [comment]
You mean like grab the prop?

**Comment by Pbtm999** [comment]
I could work on that even it seems a little strange because of the model i use

**Comment by jesuisjoszi** [comment]
yes, that's why I changed it to a regular boombox but if I understand it, there's no need to add it

**Comment by Pbtm999** [comment]
You just made me thing in something i could add thanks :)

**Comment by jesuisjoszi** [comment]
okk

**Comment by jesuisjoszi** [comment]
okk
